paper/paper.md

```diff
@@ -0,0 +1,306 @@
+  network analyses, but it also eases the reuse of all outputs in the BIDS ecosystem.
+
+\textbf{Developed with openness, transparency, and good practices in mind.}
+CMP3 is published under the terms of the open source 3-Clause Berkeley Software
+  Distribution (3-Clause BSD) license, which allows unlimited modification, redistribution
+  and commercial use in source and binary forms, as long as the copyright notice is retained and the
+  license's disclaimers of warranty are maintained.
+The source code for CMP3 is hosted at
+  [https://github.com/connectomicslab/connectomemapper3](https://github.com/connectomicslab/connectomemapper3),
+  where all bugs and contributions are transparently discussed and managed through issues, and each release is
+  archived to Zenodo [@ZenodoCMP:2021].
+In case of problems, CMP3 has a dedicated forum at
+  [groups.google.com/group/cmtk-users](groups.google.com/group/cmtk-users) where a
+  community of users is active to support each other and have scientific discussions.
+To be robust to adverse code changes, versions are released through continuous integration building
+  and testing.
+Specifically, this involves testing the installation of the python package, the build of
+  the Docker and Singularity container images, and the execution of the BIDS App via the different container
+  images adopting multiple pipeline configurations, using a sample multi-modal MRI dataset [@Tourbier2020SampleDataset]
+  that has been created for this purpose.
+Doing so, we can guarantee the full functionality of each newly released version of CMP3
+  for a range of different use cases.
+More details about CMP3, the different processing steps and generated outputs together with
+  installation and usage instructions, different tutorials supporting the analysis,
+  and the interpretation of the generated outputs with popular tools, can be found in
+  the documentation ([connectome-mapper-3.readthedocs.io](connectome-mapper-3.readthedocs.io))
+  that is kept up to date with the current release and can be retrieved for older versions.
+
+# Mention
+
+CMP3 has been successfully employed in a number of methodological
+  [@Zheng2020GeometricConnectomeb] [@GlombNeuro:2020] [@GlombNet:2020] [@AkselrodHBM:2021]
+  [@RueQueraltNeuro:2021] [@PascucciNet:2021], clinical [@Carboni2019TheOutcome] 
+  [@Vorderwulbecke2020High-densityPoint] [@CarboniNeuro:2020] [@Carboni:2022], and data [@Pascucci2022]
+  research articles.
+CMP3 is also part of [`ReproNim/containers`](https://github.com/ReproNim/containers),
+  a Datalad dataset with a collection of 40 popular containerized neuroimaging research pipelines,
+  which allows one to easily include it as a subdataset within Datalad-controlled BIDS datasets,
+  and achieve fully reproducible analysis by running CMP3 directly with Datalad.
+
+# Acknowledgements
+
+This work was supported by Swiss National Science Foundation Sinergia
+  [grant no. 170873](https://p3.snf.ch/project-170873).
```

Comment:
Hi @sebastientourbier could you add the BBP funding statement as well?

This work was supported by Swiss National Science Foundation Sinergia
  [grant no. 170873](https://p3.snf.ch/project-170873).  M. Anıl Tuncel's involvement was supported by funding to the Blue Brain Project, a research center of the École polytechnique fédérale de Lausanne (EPFL), from the Swiss government’s ETH Board of the Swiss Federal Institutes of Technology.

Reply:
Done 👍